Update Music Assistant with Bronze docs requirements

OzGav opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
It is not described anywhere how to use the browse media action with Music Assistant


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced a media browsing capability in the Music Assistant integration, allowing users to explore their media library by selecting an artist or album.

- **Documentation**
  - Updated documentation to clarify the `media_player.browse_media` action and revised details for the `media_player.play_media` action, including new URI requirements and integration removal instructions.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

OzGav committed:
Add browse media section

OzGav commented:
@marcelveldt Please advise if this is ok!

OzGav committed:
Code rabbit suggestions

OzGav committed:
Update music_assistant.markdown

OzGav committed:
code rabbit punctuation

frenck commented on source/_integrations/music_assistant.markdown:
````diff
@@ -69,6 +69,38 @@ media_content_type: playlist
 media_content_id: 'The Best of Disco'
 ```
 
+### Action `media_player.browse_media`
````
This is a media player action, not specific to music assistant.

It thus isn't integration specific and thus shouldn't be documented here but in the media player documentation.

OzGav replied:
But the media player documentation says that the media_content_id is “integration specific” so this section is intended to show the specifics?

marcelveldt replied:
browse_media is a (global) action of the media_player platform and not specific to MA, so it shouldn't be documented here, it's already documented in the media_player docs.

If you want to have something in the documentation about the supported media content type's and media content id(s), that would maybe better be fit in a notes section ?

Or maybe some specific section where it says you can use all the default media player actions from HA and when then describe the fact that MA supports the media_content_id in uri format.

OzGav replied:
All text pertaining to HA actions removed. media_content_id info added to the top.

OzGav committed:
Merge branch 'home-assistant:current' into patch-2

OzGav committed:
Amend docs to Bronze standard

Add integration removal section.
Remove HA media player actions
Add valid media_content_id info

OzGav committed:
Code rabbit suggestions

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
````diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
-media_content_type: artist
-media_content_id: 'Stevie Wonder'
-```
+After deleting the integration, go to the HA SETTINGS >> ADD-ONS and remove the Music Assistant Add-on from there as well (if installed).
````
```suggestion
After deleting the integration, go to the {% my supervisor_addon title="**Settings** > **Add-ons**" addon="core_samba" %} and remove the **Music Assistant** add-on from there as well (if installed).
```

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
+Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the HA or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed, then follow these [installati ...
 
-Play Adele's album 25
+## Removing the integration
```
I would move the removal instructions to the end of the page, as it marks the end of the integration's life-cycle.

OzGav replied:
Thanks done!

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
+### Manual configuration
 
-#### Examples
+Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the HA or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed, then follow these [installati ...
```
```suggestion
Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the Home Assistant or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed, then follow these [installation instructions](https://music-assistant.io/installation/).
```

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
+The `media_content_id` payload for `media_player.play_media` can be any of the following:
 
-The Music Assistant integration creates media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
+- The name of a track, artist or album. (for example, `Queen`)
```
```suggestion
- The name of a track, artist, or album. For example, `Queen`.
```

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
 
-The Music Assistant integration creates media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
+- The name of a track, artist or album. (for example, `Queen`)
+- A track or album combined with the artist name (for example, `Queen - Innuendo`)
```
```suggestion
- A track or album combined with the artist's name. For example, `Queen - Innuendo`.
```

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
-The Music Assistant integration creates media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
+- The name of a track, artist or album. (for example, `Queen`)
+- A track or album combined with the artist name (for example, `Queen - Innuendo`)
+- A streaming provider URI (for example, `spotify://artist/12345`)
```
```suggestion
- A streaming provider URI. For example, `spotify://artist/12345`.
```

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
+The `media_content_id` payload for `media_player.browse_media` must be a URI of the form `library://artist/1`, `library://album/20`, or `spotify://album/5zj4Ej0FrlJQaSo0d6cttH`. The type of item that the URI refers to must be an album or artist.
 
-Play media hosted on a Music Assistant server on a Music Assistant player. The action configuration is as described in the [media player documentation](https://www.home-assistant.io/integrations/media_player/#action-media_playerplay_media)
+These URIs can be obtained from, for example, the output of the `get_library` or `search` actions described below or the `media_player.browse_media` HA action. 
```
```suggestion
These URIs can be obtained from, for example, the output of the `get_library` or `search` actions described below or the `media_player.browse_media` action from Home Assistant. 
```

c0ffeeca7 committed:
tiny tweaks

OzGav committed:
Merge branch 'current' into patch-2

OzGav committed:
Review suggestions

OzGav committed:
Merge branch 'current' into patch-2

c0ffeeca7 commented on source/_integrations/music_assistant.markdown:
```diff
@@ -221,3 +195,11 @@ Get the queue details of a Music Assistant player queue.
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
```
```suggestion
This integration follows standard integration removal.
```

OzGav committed:
Update source/_integrations/music_assistant.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 reviewed (approved):
Nice addition to the docs! Thank you, @OzGav :+1:
